packages/ocular/src/api/routes/admin/apps/list-installed-apps.ts

```diff
@@ -1,17 +1,11 @@
-import { IsNotEmpty, IsString, IsEnum} from "class-validator"
-import { OrganisationService } from "../../../../services"
-import { validator } from "@ocular/utils"
-import {AppNameDefinitions} from "@ocular/types"
+import { OAuthService } from "../../../../services";
 
 export default async (req, res) => {
-  console.log("req.body  code ->", req.body)
-  const organisationService: OrganisationService = req.scope.resolve("organisationService")
-  const org = await organisationService.listInstalledApps()
-  if (!org) {
-    return res.status(404).json({ message: "No organisation found" })
-  }
-  if (!org.installed_apps) {
-    return res.status(200).json({ apps: []})
+  const oauthService: OAuthService = req.scope.resolve("oauthService");
+  const appList = await oauthService.list({});
+
+  if (!appList) {
+    return res.status(200).json({ apps: [] });
```

Comment:
return res.status(404).json({ message: "No organisation found" })

Reply:
it might happen that organization exists but they have not installed any apps yet , still send this message ?

`return res.status(404).json({ message: "No organisation found" })`

Reply:
Makes sense!

Reply:
So keeping the current code as it is , should I push changes ?